base/binaryplatforms.jl

```diff
@@ -904,7 +904,7 @@ function detect_cxxstring_abi()
     end
 
     function open_libllvm(f::Function)
-        for lib_name in ("libLLVM-14jl", "libLLVM", "LLVM", "libLLVMSupport")
+        for lib_name in (libllvm_name, "libLLVM", "LLVM", "libLLVMSupport")
```

Comment:
Should it be only libllvm_name?

Reply:
I will forward that question to @staticfloat

Reply:
Yeah, I think we've now successfully managed to abstract away the platform differences here.